main.cpp

```diff
@@ -32,24 +32,52 @@ int main(int argc, char* argv[])
     cu::fs::path input_struc_path;
     cu::fs::path input_mat_path;
     cu::fs::path output_struc_path;
+    std::vector<double> rotation_axis_input;
+    double rotation_angle_input=0;
```

Comment:
Is there a reason this gets initialized to zero? Also just for consistency I think it's better to choose either `*_input` or `input_*` for the variables.